wandb/sdk/wandb_run.py

```diff
@@ -138,6 +138,12 @@ class TeardownHook(NamedTuple):
     stage: TeardownStage
 
 
+class WaitTimeoutError(Exception):
```

Comment:
Do you think it would be good idea to move this to `errors/__init__.py`

Reply:
moved WaitTImeoutError to errors/__init__.py